setup.py

```diff
@@ -1,135 +0,0 @@
-"""Setup for bcipy package.
```

Comment:
This needs to be put back! We need this to install and release our code

Reply:
Sorry, didn't realize it was removed! It must be done accidentally during 2.0.0rc4 branch merge.